Djikstra/Djikstra/Functions.cpp

```diff
@@ -0,0 +1,135 @@
+#include <vector>
+
+
+#include "Headers.h"
+
+
+namespace Graph
+{
+	bool Graph::node_exist( std::string name)
+	{
+		int i=0, vector_size;
```

Comment:
You can do all the job of this function just exercising [`std::map<string, int>`](https://en.cppreference.com/w/cpp/container/map) (remember to `#include <map>`). Since we are not using C++20 yet, you will have to do something like [`m_names_to_ids.find(name) != m_names_to_ids.end()`](https://en.cppreference.com/w/cpp/container/map/find).